src/main/java/Currencies.java

```diff
@@ -1,12 +1,21 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
 public enum Currencies {
-    EUR(new int[]{500, 200, 100, 50, 20, 10, 5, 2, 1}),
-    HUF(new int[]{20000, 10000, 5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 5});
+    EUR(new int[]{200, 100, 50, 20, 10, 5, 2, 1, 500}),
+    HUF(new int[]{2000,20000, 10000, 5000, 1000, 500, 200, 100, 50, 20, 10, 5});
 
-    public int[] bankNotes;
+    private int[] bankNotes;
 
     Currencies(int[] bankNotes) {
         this.bankNotes = bankNotes;
     }
 
-
+    public ArrayList<Integer> getBankNotesSortedDesc() {
```

Comment:
A bit too complicated.
If you need List<>, then you could use https://www.geeksforgeeks.org/arrays-aslist-method-in-java-with-examples/
Then it is possible to simply: https://www.geeksforgeeks.org/collections-sort-java-examples/